ui/packages/catalog-ui-search/src/main/webapp/react-component/container/loading-companion/loading-companion.tsx

```diff
@@ -31,9 +31,9 @@ export default class LoadingCompanionContainer extends React.Component<
   }
   ref = React.createRef()
   componentDidUpdate() {
-    if (this.props.loading === false) {
-      LoadingCompanionView.stopLoadingElement(this.ref.current)
-    }
+    this.props.loading === false
```

Comment:
✏️ 
```suggestion
    !this.props.loading
```

Reply:
I'd go even further and say you should just swap the ternary and leave as `this.props.loading`